preview/Win7Msix/Win7MSIXInstaller/GeneralUtil.cpp

```diff
@@ -52,4 +52,14 @@ std::wstring GetStringResource(UINT resourceId)
     std::wstring stringResource(buffer);
 
     return stringResource;
+}
+
+std::wstring GetFamilyNameFromFullName(const std::wstring& fullName)

```

Comment:
>GetFamilyNameFromFullName [](start = 13, length = 25)

Add comments

Reply:
there are comments to this function in the header. i'm going to move all function-level comments in this file to the header

---
In reply to: [269348006](https://github.com/Microsoft/msix-packaging/pull/58#discussion_r269348006) [](ancestors = 269348006)